### PR TITLE
♻️ Re-style buttons based on guidelines

### LINKED ISSUE #68: Refactor button styles
The buttons should be refactored to limit the number of colors and styles available to match the UI Guidelines.
screen shot 2018-10-30 at 6 44 27 pm
screen shot 2018-10-29 at 1 06 12 pm
screen shot 2018-10-29 at 1 06 15 pm
screen shot 2018-10-29 at 1 06 18 pm

### PR DESCRIPTION
**Pull request by dankolbman**
Formats the buttons according to the UI Guidelines.

### Defaults

The default button style is now the grey border with teal text.

new_buttons_1

### Removal of 'outline' style

The 'outline' style has been removed. Any legacy components using it will become the equivalent filled style.

new_buttons_1

### Removal of 'small' size

The small sized button has been removed. Existing usage will be transformed into the default size.

Fixes #68

### CODE REVIEW

**Comment by dankolbman** [comment]
Needs margin!

screen shot 2018-10-31 at 10 21 12 am

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -30,11 +30,9 @@ const Button = ({ size, color, outline, disabled, onClick, type, className,
 
 Button.propTypes = {
   /** The size of the button */
-  size: propTypes.string,
+  size: propTypes.oneOf(['default', 'large']),
   /** The color of the button. */
-  color: propTypes.string,
-  /** Whether the button is an outline only, or solid. */
-  outline: propTypes.bool,
```
Comment (bdolly) [inline src/components/Button/Button.jsx]: @dankolbman I would investigate whether there are outline buttons still in use in the wild. If there are then they need to documented by @kcullion in the ui guidelines before this gets deprecated.

File: src/components/Button/Button.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        border
+        border-greyBorder
+        rounded-lg
+        font-bold
+        font-title
+        text-xs
+        text-tealA11y
+        leading-tight;
+
+    text-transform: uppercase;
```
Comment (bdolly) [inline src/components/Button/Button.css]: why the orphan style and not a in the `@apply`?

File: src/components/Button/Button.jsx (hunk 1 above)
Comment (dankolbman) [reply src/components/Button/Button.jsx]: Updated the description with the changes. Only coordinator is affected here.

Comment (bdolly) [reply src/components/Button/Button.jsx]: when deprecating from the outline shouldn't the fill color be the same as the outline color was? In your example in the description it seems the button color changes which can be confusing to users as functionality should be encoded by color

Comment (dankolbman) [reply src/components/Button/Button.jsx]: The previous implementation used a different color as a default as the color palette naming wasn't correct. This was also corrected in this pr.

User wise, this only impacts internal users atm.

**Pull request merged**